### PR TITLE
STORM-1459: allow not specifying producer properties in read-only Kaf…

### PR DESCRIPTION
**Pull request by manuzhang**
…ka table in StormSQL

### CODE REVIEW

File: docs/storm-sql.md [hunk 1]
```diff
@@ -104,21 +104,22 @@ Let's say there is a Kafka stream that represents the transactions of orders. Ea
...
-Current implementation of `storm-sql-kafka` requires specifying both `LOCATION` and `TBLPROPERTIES` clauses even though the table is read-only or write-only.
+The first statement defines the table `ORDER` which represents the input stream. The `LOCATION` clause specifies the ZkHost (`localhost:2181`), the path of the brokers in ZooKeeper (`/brokers`) and the topic (`orders`). 
 
-Similarly, the second statement specifies the table `LARGE_ORDERS` which represents the output stream. The third statement is a `SELECT` statement which defines the topology: it instructs StormSQL to filter all orders in the external table `ORDERS`, calculates the total price and inserts matching records into the Kafka stream specified by `LARGE_ORDER`.
+Similarly, the second statement specifies the table `LARGE_ORDERS` which represents the output stream. The `TBLPROPERTIES` clause specifies the configuration of [KafkaProducer](http://kafka.apache.org/documentation.html#producerconfigs).
```
Comment (HeartSaVioR) [inline docs/storm-sql.md]: Might be better to clarify again that `TBLPROPERTIES` is optional for table as a producer (read) and mandatory for table as a consumer (write).

**Comment by HeartSaVioR** [comment]
+1 except minor documentation issue.

**Comment by manuzhang** [comment]
@HeartSaVioR Updated.  I use "Kafka sink table" instead of "consumer" to avoid confusing with kafka consumer.  And I don't think `TBLPROPERTIES` is an option for "table as a producer" so I leave it out.

**Comment by HeartSaVioR** [comment]
Great. Thanks for the quick update. +1

**Pull request merged**